app/models/ReactiveAccount.php

```diff
@@ -4,7 +4,7 @@
 class ReactiveAccount extends Model{
```

Comment:
Change the name of the file also
```suggestion
class Activation extends Model
```